### ISSUE TITLE
[Chromaticraft] [Server] "BlockStructureShield cannot be cast to" light tape puzzle thing 

### ISSUE THREAD

**Issue by lAimBotl** [ReikaKalseki/Reika_Mods_Issues#2848]
https://pastebin.com/4Fqz7wyg

Happened randomly when I pressed play on the tape thingy, in the CRC dim.

**Comment by lAimBotl** [comment]
found a way to possibly repeat it ( I am not actually sure yet if it is repeatable this way (takes me a while to find that puzzle again :s ), but thought I would post what I did in more detail this time, since I was actually paying attention now lel), use the bottom thingy then click the middle "play" thing (not the top one) a bunch till  you get back to the start, then do it one more time, eg: bottom, mid to test, then do that through the whole tape till you are back at the start, then do it again.

**Comment by ReikaKalseki** [comment]
So basically spamming the input?

**Comment by lAimBotl** [comment]
not really, I was just tryin to get everything right before I clicked the "play" on it (I like to double check stuff lol) though spamming might work too, guess I will try to test that too

**Comment by lAimBotl** [comment]
I also noticed that it seems like sometimes the button doesn't move the tape properly, would be easier to explain with a video I think . . .
